core/src/main/java/com/threerings/getdown/util/Config.java

```diff
@@ -237,7 +237,7 @@ public String getString (String name) {
      */
     public String getString (String name, String def) {
         String value = (String)_data.get(name);
-        return value == null ? def : value;
+        return StringUtil.isBlank(value) ? def : value;
```

Comment:
This is another behavioral change that I don't want to see in an unrelated PR.

Reply:
that change is somewhat related. look at the place where _javaLocalDir is initialized. that's just a defence against empy values. but ok, i'll revert and make the check in that place